src/EnergyPlus/WaterUse.hh

```diff
@@ -221,7 +222,7 @@ namespace WaterUse {
               ColdSupplyTemp(0.0), ColdTemp(0.0), HotTemp(0.0), DrainTemp(0.0), RecoveryTemp(0.0), ReturnTemp(0.0), WasteTemp(0.0), TempError(0.0),
               MainsVolume(0.0), TankVolume(0.0), ColdVolume(0.0), HotVolume(0.0), TotalVolume(0.0), Power(0.0), Energy(0.0), NumWaterEquipment(0),
               MaxIterationsErrorIndex(0), PlantLoopNum(0), PlantLoopSide(0), PlantLoopBranchNum(0), PlantLoopCompNum(0), MyEnvrnFlag(true),
-              setupMyOutputVars(true)
+              setupMyOutputVars(true), plantScanFlag(true)
```

Comment:
The flag needed to be added as a member variable.  I could have potentially hijacked the `setupMyOutputVars` flag, but then it would get sketchy as that flag would be doing more than just setting up output variables.  Keeping them separate is good for this use case.